File: back/src/main/java/fr/isep/genielogiciel/entities/Exam.java
package fr.isep.genielogiciel.entities;

import lombok.*;
import org.springframework.data.mongodb.core.mapping.Document;

import java.util.List;

@Getter @Setter @Builder
@NoArgsConstructor
@AllArgsConstructor
@Document(collection = "exams")
public class Exam {

    private String id;
    private List<String> age;
    private String bigDescription;
    private List<String> countryList;
    private String description;
    private String familyBackground;
    private boolean isVaccine;
    private String name;
    private String personalBackground;
    private List<String> recommendation;
    private List<String> sex;

}

File: front/src/app/data/exam.ts
export interface ExamListItem{

    id : string;
    name : string;
    age : number[];
    bigDescription: string;
    countryList: string[];
    description: string;
    isVaccine: boolean;
    recommendation?: string[];
    familyBackground?: string;
    personalBackground?: string;
    sex: string[];

    isExpanded: boolean;
}